dstep/Configuration.d

```diff
@@ -77,6 +77,10 @@ struct Configuration
     /// do not translate following symbols
     @("skip", "Skip translation of <symbol>.")
     string[] skipSymbols;
+
+    /// enable printing of warnings
+    @("enable-warnings", "Enable printing of warnings [default].")
+    bool enableWarnings = true;
```

Comment:
Hmm, we might not need different levels. `--print-diagnostics` would be a boolean option. Then (at a later time) we could add `--diagnostic-action` which could be `ignore`, `abort`, `try-fix` and so on.